nvm.sh

```diff
@@ -335,21 +335,40 @@ nvm_find_nvmrc() {
   fi
 }
 
-# Obtain nvm version from rc file
+# Obtain the nvm version from the .nvmrc file
 nvm_rc_version() {
+  # Check if quiet mode is on
+  local quiet="false"
+  if [ "$1" == "quiet" ]; then
```

Comment:
```suggestion
  if [ "$1" == "--silent" ]; then
```
for consistency. or even simpler, just pass `NVM_USE_SILENT` instead of an arg?